test/cpp/interop/interop_client.h

```diff
@@ -146,21 +151,26 @@ class InteropClient {
   std::tuple<bool, int32_t, std::string, std::string>
   PerformOneSoakTestIteration(
       const bool reset_channel,
-      const int32_t max_acceptable_per_iteration_latency_ms);
+      const int32_t max_acceptable_per_iteration_latency_ms,
+      const int32_t request_size, const int32_t response_size);
 
   void PerformSoakTest(const std::string& server_uri,
                        const bool reset_channel_per_iteration,
                        const int32_t soak_iterations,
                        const int32_t max_failures,
                        const int32_t max_acceptable_per_iteration_latency_ms,
                        const int32_t min_time_ms_between_rpcs,
-                       const int32_t overall_timeout_seconds);
+                       const int32_t overall_timeout_seconds,
+                       const int32_t request_size, const int32_t response_size);
 
   ServiceStub serviceStub_;
   /// If true, abort() is not called for transient failures
   bool do_not_abort_on_transient_failures_;
   // Load Orca metrics captured by the custom LB policy.
   LoadReportTracker load_report_tracker_;
+
+  static constexpr int kLargeRequestSize = 271828;
```

Comment:
Do these constants redefined here? If not can we leave them as they were next to the other constants?

Reply:
Moved it back. PTAL.